ee/clickhouse/client.py

```diff
@@ -135,7 +135,7 @@ def cache_sync_execute(query, args=None, redis_client=None, ttl=CACHE_TTL, setti
             redis_client.set(key, _serialize(result), ex=ttl)
             return result
 
-    def sync_execute(query, args=None, settings=None, with_column_types=False):
+    def sync_execute(query, args=None, settings=None, with_column_types=False, as_dict=False):
```

Comment:
Hmm. This muddies the return type of this base function a lot - it doesn't feel like we want even more options here.

Let's discuss this through during pairing.